File: year2020/month09/day0910/ms_0910.cpp
#include <iostream>
#include <string>

using namespace std;

void print();

int main() {
	string data[3] = { "first", "second", "third" };

	for (int i = 0; i < 3; i++) {
		cout << data[i] << endl;
		print();
	}

	return 0;
}

void print() {
	cout << "@@@@@@@@@@" << endl;
}

File: year2020/month09/day0910/ms_0910.cs
using System;

namespace algorithm_54
{
    class ms_0910
    {
        static void print() 
        {
            Console.WriteLine("@@@@@@@@@@");
        }

        static void Main(string[] args)
        {
            string[] data = { "first", "second", "third" };

            for(int i = 0; i < 3; i++) 
            {
                Console.WriteLine(data[i]);
                print();
            }
        }
    }
}
